### PR TITLE
🌐 Add Ukrainian translation for `docs/uk/docs/tutorial/body.md`

### PR DESCRIPTION
**Pull request by ss-o-furda**
related: https://github.com/tiangolo/fastapi/issues/1748

### CODE REVIEW

**Review by khlopko — APPROVED** [review]
Translated text exactly matches the content in English documentation and correct in Ukranian language.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 8bcc2659abd9a55364ff72a4bd10c657517c5247 at: https://649a13994baeac0f81ee7234--fastapi.netlify.app

File: docs/uk/docs/tutorial/body.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+Ваш API майже завжди має надсилати тіло **відповіді**. Але клієнтам не обов’язково потрібно постійно надсилати тіла **запитів**.
+
+Щоб оголосити тіло **запиту**, ви використовуєте <a href="https://pydantic-docs.helpmanual.io/" class="external-link" target="_blank">Pydantic</a> моделі з усією їх потужністю та перевагами.
+
+!!! info
+    Щоб надіслати дані, ви повинні використовувати один із: `POST` (більш поширений), `PUT`, `DELETE` або `PATCH`.
+
+    Надсилання тіла із запитом `GET` має невизначену поведінку в специфікаціях, проте воно підтримується FastAPI лише для дуже складних/екстремальних випадків використання.
+
+    Оскільки це не рекомендується, інтерактивна документація з Swagger UI не відображатиме документацію для тіла запиту під час використання `GET`, і проксі-сервери в середині можуть не підтримувати її.
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/body.md:16]:
```suggestion
    Оскільки це не рекомендується, інтерактивна документація з Swagger UI не відображатиме документацію для тіла запиту під час використання `GET`, і проксі-сервери вcередині можуть не підтримувати її.
```

File: docs/uk/docs/tutorial/body.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+    {!> ../../../docs_src/body/tutorial001.py!}
+    ```
+
+=== "Python 3.10 і вище"
+
+    ```Python hl_lines="5-9"
+    {!> ../../../docs_src/body/tutorial001_py310.py!}
+    ```
+
+Так само, як і при оголошенні параметрів запиту, коли атрибут моделі має значення за замовчуванням, він не є обов’язковим. В іншому випадку це потрібно. Використовуйте `None`, щоб зробити його необов'язковим.
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/body.md:52]:
```suggestion
Так само, як і при оголошенні параметрів запиту, коли атрибут моделі має значення за замовчуванням, він не є обов’язковим. В іншому випадку - він потрібен. Використовуйте `None`, щоб зробити його необов'язковим.
```

File: docs/uk/docs/tutorial/body.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+...оскільки `description` і `tax` є необов'язковими (зі значенням за замовчуванням `None`), цей JSON "`об'єкт`" також буде дійсним:
+
+```JSON
+{
+    "name": "Foo",
+    "price": 45.2
+}
+```
+
+## Оголоси її як параметр
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/body.md:74]:
```suggestion
## Оголосіть її як параметр
```

File: docs/uk/docs/tutorial/body.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+* Валідувати дані.
+    * Якщо дані недійсні, він поверне гарну та чітку помилку, вказуючи, де саме і які дані були неправильними.
+* Надавати отримані дані у параметрі `item`.
+    * Оскільки ви оголосили його у функції як тип `Item`, ви також матимете всю підтримку редактора (автозаповнення, тощо) для всіх атрибутів та їх типів.
+* Генерувати <a href="https://json-schema.org" class="external-link" target="_blank">JSON Schema</a> визначення для вашої моделі, ви також можете використовувати їх де завгодно, якщо це має сенс для вашого проекту.
+* Ці схеми будуть частиною згенерованої схеми OpenAPI і використовуватимуться автоматичною документацією інтерфейсу користувача.
+
+## Автоматична документація
+
+Схеми JSON ваших моделей будуть частиною вашої схеми, згенерованої OpenAPI, і будуть показані в інтерактивній API документації:
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/body.md:107]:
```suggestion
Схеми JSON ваших моделей будуть частиною вашої схеми, згенерованої OpenAPI, і відображатимуться в інтерактивній API документації:
```

File: docs/uk/docs/tutorial/body.md [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+
+<img src="/img/tutorial/body/image01.png">
+
+А також використовуватимуться в API документації всередині кожної *операції шляху*, якій вони потрібні:
+
+<img src="/img/tutorial/body/image02.png">
+
+## Підтримка редактора
+
+У вашому редакторі, всередині вашої функції, ви будете отримувати підказки типу та завершення скрізь (це б не сталося, якби ви отримали `dict` замість моделі Pydantic):
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/body.md:117]:
```suggestion
У вашому редакторі, всередині вашої функції, ви будете отримувати підказки типу та завершення скрізь (цього не сталося б, якби ви отримали `dict` замість моделі Pydantic):
```

File: docs/uk/docs/tutorial/body.md [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+
+Були навіть деякі зміни в самому Pydantic, щоб підтримати це.
+
+Попередні скріншоти були зроблені у <a href="https://code.visualstudio.com" class="external-link" target="_blank">Visual Studio Code</a>.
+
+Але ви отримаєте ту саму підтримку редактора у <a href="https://www.jetbrains.com/pycharm/" class="external-link" target="_blank">PyCharm</a> та більшість інших редакторів Python:
+
+<img src="/img/tutorial/body/image05.png">
+
+!!! tip
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/body.md:137]:
```suggestion
!!! підказка
```

File: docs/uk/docs/tutorial/body.md [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+    {!> ../../../docs_src/body/tutorial004_py310.py!}
+    ```
+
+Параметри функції будуть розпізнаватися наступним чином:
+
+* Якщо параметр також оголошено в **шляху**, він використовуватиметься як параметр шляху.
+* Якщо параметр має **сингулярний тип** (наприклад, `int`, `float`, `str`, `bool` тощо), він буде інтерпретуватися як параметр **запиту**.
+* Якщо параметр оголошується як тип **Pydantic моделі**, він інтерпретується як **тіло** запиту.
+
+!!! note
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/body.md:206]:
```suggestion
!!! примітка
```

File: docs/uk/docs/tutorial/body.md [hunk 8]
```diff
@@ -0,0 +1,213 @@
...
+    FastAPI буде знати, що значення "q" не є обов'язковим через значення за замовчуванням "= None".
+
+    `Optional` у `Optional[str]` не використовується FastAPI, але дозволить вашому редактору надати вам кращу підтримку та виявляти помилки.
+
+## Без Pydantic
+
+Якщо ви не хочете використовувати моделі Pydantic, ви також можете використовувати параметри **Body**. Перегляньте документацію для [Тіло – Кілька параметрів: сингулярні значення в тілі](body-multiple-params.md#singular-values-in-body){.internal-link target=_blank}.
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/body.md:213]:
```suggestion
Якщо ви не хочете використовувати моделі Pydantic, ви також можете використовувати параметри **Тіла**. Перегляньте документацію для [Тіло – Кілька параметрів: сингулярні значення в тілі](body-multiple-params.md#singular-values-in-body){.internal-link target=_blank}.
```

**Review by romashevchenko — APPROVED** [review]
Hi there! I'm a native speaker. I've reviewed the translation and made several suggestions to improve it.

**Comment by tiangolo** [comment]
Great, thanks @ss-o-furda! 🍰 

And thanks for the reviews @khlopko and @romashevchenko 🙇

**Pull request merged**